test/case/ietf_interfaces/iface_enable_disable/Readme.adoc

```diff
@@ -0,0 +1,28 @@
+=== Interface status
+==== Description
+Verify that interface status works
+
```

Comment:
I guess we should update the generated files after that description changes.

Reply:
Oh yes. I still forget that :( ....

Reply:
Coming